_pytest/debugging.py

```diff
@@ -85,6 +85,12 @@ def _enter_pdb(node, excinfo, rep):
     # for not completely clear reasons.
     tw = node.config.pluginmanager.getplugin("terminalreporter")._tw
     tw.line()
+
+    captured_logs = rep.caplog
+    if len(captured_logs) > 0:
```

Comment:
You should also check if live-logging is enabled here, in which case it does not make sense to print the captured logs again.

Reply:
I agree with @brianmaissy that just printing them here is more consistent, but I don't have strong feeling either way to be honest.

Reply:
@thisch what do you think? If you agree we can merge this. 👍

Reply:
I think we can merge this. But I definitely want to change this "before entering pdb" behavior when capturing is off (`-s`) and/or live logging is enabled. I'll create a new issue for this with some screenshots.